build/buildBuildImages.sh

```diff
@@ -415,11 +454,12 @@ if [ -z "$imageTypeToBuild" ]; then
 	buildJamStackImage
 	buildLtsVersionsImage "buster"
 	buildLtsVersionsImage	
-	buildFullImage
+	buildlatestImages
```

Comment:
Nit: looks like an uppercase is missing. Not sure if that would create an error?

Reply:
This would. Updated. Thanks :)